public/app/core/time_series2.ts

```diff
@@ -70,6 +70,8 @@ export function getDataMinMax(data: TimeSeries[]) {
 export default class TimeSeries {
   datapoints: any;
   id: string;
+  dataFrameIndex: number;
+  fieldName: string;
```

Comment:
Can we add some comments to these fields that say why they are needed and maybe a comment saying this whole class is deprecated and should not be used in any new panels?